tests/core/engine_adapter/test_trino.py

```diff
@@ -89,32 +95,37 @@ def mock_fetchone(sql):
     assert fetchone_mock.call_count == 2
 
 
-def test_partitioned_by_hive(
+@pytest.mark.trino_delta
+def test_partitioned_by_hive_delta(
     trino_mocked_engine_adapter: TrinoEngineAdapter, mocker: MockerFixture
 ):
     adapter = trino_mocked_engine_adapter
 
-    mocker.patch(
-        "sqlmesh.core.engine_adapter.trino.TrinoEngineAdapter.get_current_catalog",
-        return_value="datalake_hive",
-    )
-    assert adapter.get_catalog_type("datalake_hive") == "hive"
+    storage_types = ("hive", "delta")
 
-    columns_to_types = {
-        "cola": exp.DataType.build("INT"),
-        "colb": exp.DataType.build("TEXT"),
-    }
+    for storage_type in storage_types:
```

Comment:
What do you think about doing this with `@pytest.mark.parametrize(` having delta and hive be parameters? That would remove this inner loop and allow the test to assert just two commands are run instead of 2 * 2. Same applies to other tests where you have this inner loop over storage types.